### PR TITLE
📖  Document the WSL-helm-ghcr fail and workaround

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR updates the website to document the trouble and workaround reported in Issue #2544 . This is an alternative to PR #2519 .

Preview at https://mikespreitzer.github.io/kcp-edge-mc/doc-wsl-fail/direct/knownissue-wsl-ghcr-helm/
and https://mikespreitzer.github.io/kcp-edge-mc/doc-wsl-fail/direct/get-started/#note-for-windows-users

### CODE REVIEW

File: docs/content/direct/get-started.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ This page shows one concrete example of steps 2--7 from the [full Installation a
 
 This is one way to produce a very simple system, suitable for study but not production usage. For general setup information, see [the full story](user-guide-intro.md#the-full-story).
 
+### Note for Windows users
+
+Someone using WSL [had a problem with the recipe on this page and was able to work around it by running the whole recipe as the `root` user in Linux](knownissue-wsl-ghcr-helm.md).
```
Comment (clubanderson) [inline docs/content/direct/get-started.md:19]: 'someone'? Why are we pointing out a person in our documentation? 

"For some users on WSL, use of our getting started guide and demo environment creation script may require running with the `root` user."

File: docs/content/direct/knownissue-wsl-ghcr-helm.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Authorization failure in WSL while fetching Helm chart from ghcr.io
+
+## Description of the Issue
+
+Someone using WSL (Windows Subsystem for Linxu) and following the
```
Comment (clubanderson) [inline docs/content/direct/knownissue-wsl-ghcr-helm.md:5]: again - 'someone'?

**Comment by clubanderson** [comment]
'someone' misspelled a word in the title of this PR

**Comment by MikeSpreitzer** [comment]
The force-push to c5659eeee does some rewording in response to the review above.

File: docs/content/direct/get-started.md (hunk 1 above)
Comment (MikeSpreitzer) [reply docs/content/direct/get-started.md]: Reworded.

File: docs/content/direct/knownissue-wsl-ghcr-helm.md (hunk 2 above)
Comment (MikeSpreitzer) [reply docs/content/direct/knownissue-wsl-ghcr-helm.md]: Reworded.

**Comment by francostellari** [comment]
Given the little information around this problem, I'm of the opinion that 
1. this issue does not have anything to do with KS but rather helm
2. it should not be included in the get started procedure 
3. it is ok to include in the _un_known issues

**Review by clubanderson — COMMENTED** [review]
/lgtm

**Comment by KPRoche** [comment]
We already have a link to troubleshooting on the get-started page, and the page linked there has a link to known issues, but the Windows known issue isn't on that list...
So as is, the aside in the introductory paragraph  adds a bit of a tangle to our troubleshooting info, but it is easily fixed if the Windows known issue were added to the list of known issues. (Note: I'm not advocating that the aside be removed)

**Comment by MikeSpreitzer** [comment]
The force-push to 1cb22729b corrects the oversight that @KPRoche noted in his review above.

**Pull request merged**